src/routes/(app)/_components/layout/navbar.svelte

```diff
@@ -1,5 +1,16 @@
 <script>
+  // @ts-expect-error Import is as expected but throws error
+
   import { onMount } from 'svelte';
+  import { page } from '$app/stores';
```

Comment:
```suggestion
  import { onMount } from 'svelte';
    // @ts-expect-error Import is as expected but throws error
  import { page } from '$app/stores';
```
That comment needs to be above the line with the error for it to work :)

Reply:
done!